kafka-streams-quickstart/.gitignore

```diff
@@ -0,0 +1,35 @@
+# Eclipse
+.project
+.classpath
+.settings/
+bin/
+
+# IntelliJ
+.idea
+*.ipr
+*.iml
+*.iws
+
+# NetBeans
+nb-configuration.xml
+
+# Visual Studio Code
+.vscode
+
+# OSX
+.DS_Store
+
+# Vim
+*.swp
+*.swo
+
+# patch
+*.orig
+*.rej
+
+# Maven
+target/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+release.properties
```

Comment:
Is there any reason why you added that in a subdirectory? Shouldn't we just have a global `.gitignore`?

Reply:
It was generated by Quarkus :) I'll remove it.

Reply:
It was generated by Quarkus :) I'll remove it.

Reply:
It was generated by Quarkus :) I'll remove it.